packages/story-editor/src/components/library/panes/shopping/shoppingPane.js

```diff
@@ -173,12 +170,12 @@ function ShoppingPane(props) {
   }, [isShoppingEnabled]);
 
   useEffect(() => {
-    if (!isShoppingEnabled) {
+    if (!isShoppingEnabled && products.length === 0) {
```

Comment:
Added check for `products.length` here

When I was testing this was seeing issues with state updates in the useEffect

![Screen Shot 2022-06-27 at 6 44 33 AM](https://user-images.githubusercontent.com/62242/175926857-3d30293f-a822-4ea9-abcf-2052dc0edad5.png)